modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordLogsService.kt

```diff
@@ -0,0 +1,72 @@
+package com.metalbear.mirrord
+
+import com.intellij.execution.filters.TextConsoleBuilderFactory
+import com.intellij.execution.ui.ConsoleView
+import com.intellij.execution.ui.ConsoleViewContentType
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.components.Service
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.wm.ToolWindowManager
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import javax.swing.JComponent
+
+@Service(Service.Level.PROJECT)
+class MirrordLogsService(private val project: Project) {
+    private var consoleView: ConsoleView? = null
```

Comment:
Why can this be `null`?

Reply:
So the consoleView can be recreated if it's disposed.